setup_proxmox.sh

```
@@ -0,0 +1,630 @@
+#!/bin/bash
```

Shellcheck (0.10.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)